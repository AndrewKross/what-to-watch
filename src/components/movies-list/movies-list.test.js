import React from "react";
import renderer from "react-test-renderer";
import MoviesList from "./movies-list.jsx";

it(`Should MoviesList render correctly`, () => {
  const tree = renderer
    .create(<MoviesList
      moviesList = {[{
        title: `Revenant`,
        src: `./img/revenant.jpg`,
        id: `1`,
      },
      {
        title: `War of the Worlds`,
        src: `./img/war-of-the-worlds.jpg`,
        id: `2`,
      },
      {
        title: `Snatch`,
        src: `./img/snatch.jpg`,
        id: `3`,
      }]}
      onMovieTitleClick = {() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
