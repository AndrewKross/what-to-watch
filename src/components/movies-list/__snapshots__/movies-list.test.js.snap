// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MoviesList render correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="\${movieData.title}"
        height="175"
        src="./img/revenant.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
      onClick={[Function]}
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Revenant
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="\${movieData.title}"
        height="175"
        src="./img/war-of-the-worlds.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
      onClick={[Function]}
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        War of the Worlds
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="\${movieData.title}"
        height="175"
        src="./img/snatch.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
      onClick={[Function]}
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Snatch
      </a>
    </h3>
  </article>
</div>
`;
